go/internal/discordlog/discord.go

```diff
@@ -0,0 +1,61 @@
+package discordlog
+
+import (
+	"encoding/base64"
+	"strings"
+
+	"berty.tech/berty/v2/go/pkg/errcode"
+	"github.com/itsTurnip/dishooks"
+	qrcode "github.com/skip2/go-qrcode"
+)
+
+// rooms are webhook URLs encoded in base64 to prevent most stupid bots to spam our channels.
+//
+// if we get spammed, then we can think about a better way to store secrets on the repo and in the releases.
+const (
+	QRCodeRoom = "aHR0cHM6Ly9kaXNjb3JkYXBwLmNvbS9hcGkvd2ViaG9va3MvNzEyNjkwNTc5NTk0ODA1Mzc5L2R2OTFFX0pqM0xkYVdFM2p5N0N1TDAtMmRzQ1NTaE85Q2RGRG05QUhtQThCWFRXS0lEZ29jUFRFRDVyTU5KVlFkcFJ5"
```

Comment:
Why don't we set this in an env variable and/or a flag?
It could be useful to avoid spam and to reuse this package with another room/webhook.

Reply:
> Why don't we set this in an env variable and/or a flag?

to avoid writing runtime doc

> It could be useful to avoid spam

if spam, we reset the webhook url and find a better approach (-> we won't be spammed)

> and to reuse this package with another room/webhook.

yep, that's why I named it QRCodeRoom in a `const(\n)` list of future potential rooms

it's a devtool, make things simple :)